docs/upgrade.html

```diff
@@ -149,6 +150,18 @@
   initial performance analysis of the new message format. You can also find more detail on the message format in the
   <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-98+-+Exactly+Once+Delivery+and+Transactional+Messaging#KIP-98-ExactlyOnceDeliveryandTransactionalMessaging-MessageFormat">KIP-98</a> proposal.
 </p>
+<p>One of the notable differences in the new message format is that even uncompressed messages are stored together as a single message set.
```

Comment:
I think we have been using `batch` instead of `message set` in the docs, so maybe we should do that here. Maybe something like "stored together as a single batch (also known as a message set)`.